ci/tests/test-financial-templates-lib.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+
+TESTS_GLOB="$(echo 'circleci tests glob "test/**/*.js"')"
+TESTS_FILE="$(echo '$(cat /tmp/test-files)')"
```

Comment:
Interesting. Just curious, why did you decide to put this inside a variable up here rather than just writing it out in the command itself?

Reply:
good catch! I've used for debug proses, and before split the test file on `ci/tests/`used  as variable over the same file, and stay as is on the new one. Fixed for now!